packages/nx-go/executors.json

```diff
@@ -10,6 +10,11 @@
       "schema": "./src/executors/lint/schema.json",
       "description": "Lint a Go program"
     },
+    "tidy": {
+      "implementation": "./src/executors/tidy/executor",
+      "schema": "./src/executors/tidy/schema.json",
+      "description": "Tidy a Go program"
```

Comment:
This tool will not tidy the Go program itself but rather the `.mod` file, right?

Reply:
It will use the app or library cwd and runs go mod tidy on the go.mod file. 
So if I take the [documentation](https://go.dev/ref/mod#go-mod-tidy) that it will update the mod file and remove unused packages of that specific module.

I tested this in my own NX project where there was a apps/nx-go-api and I ran 

```
npx nx run nx-go-api:tidy
```

It removed the unused modules from my mod file that was located at apps/nx-go-api/go.mod

Reply:
Thank you for the explanation!
It might be interesting to describe here that it will tidy the mod file and not the Go program itself.

Reply:
@utarwyn I tested it again to be 100% sure. I created a go-api application and a lib/go-api/s3. I added code for s3 inside the lib/go-api/s3 library and did a npx nx run lib/go-api/s3:tidy and it installed the packages for s3 inside the go.mod file of the lib/go-api/s3. After that I used the code inside the go-api application and did a npx nx run go-api:tidy and it did NOT install the aws-sdk dependencies in the go.mod file of the go-api which is what we wanted :).

I changed the description but don't know if it cover the whole feature. If you want it different let me know :)

Reply:
Thank you for the tests, and the description looks good now!